jlexer/lexer.go

```diff
@@ -14,6 +15,8 @@ import (
 	"unsafe"
 )
 
+var UseSemanticErrors = flag.Bool("use_many_errors", false, "Allow lexer collect semantic errors")
```

Comment:
Don't use flag.Bool here. You'll probably need to add the option as a Lexer field.

Also, UseSemanticErrors != use_many_errors. I'd go with MultipleErrors name, because it's the point: we want to report multiple errors if we are able to. The SemanticErrors name is misleading here.